src/IIIFingest/client.py

```diff
@@ -45,6 +45,8 @@ def __init__(
         environment: str = "qa",
         jwt_creds=None,
         boto_session=None,
+        ingest_endpoint=None,
+        job_endpoint=None,
```

Comment:
Currently, the `ingest_endpoint` and `job_endpoint` attributes are automatically configured in the constructor function based on the value of `environment`, but I could see a case where the user might want to manually provide those and override the defaults. Is that why they are being added to the constructor? The only reason I ask is that I don't see any other changes having to do with that or any tests.

Reply:
Ahh yes, this was an artifact from when I was trying to run functional test as to provide a mock endpoint if needed. Will remove since we aren't running any functional client test at the moment.